main.tf

```diff
@@ -1,13 +1,74 @@
-resource "random_integer" "example" {
+#-----------------------------------------------------------------------------------------------------------------------
+# Subscribe the Acccount to Security Hub
+#-----------------------------------------------------------------------------------------------------------------------
+resource "aws_securityhub_account" "this" {
   count = module.this.enabled ? 1 : 0
+}
+
+#-----------------------------------------------------------------------------------------------------------------------
+# Optionally subscribe to Security Hub Standards 
+# https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards.html
+#
+# arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0
+# arn:aws:securityhub:us-east-1::standards/aws-foundational-security-best-practices/v/1.0.0
+# arn:aws:securityhub:us-east-1::standards/pci-dss/v/3.2.1
```

Comment:
Hrm... I think the module expects only the last part, right?

```suggestion
# ruleset/cis-aws-foundations-benchmark/v/1.2.0
# standards/aws-foundational-security-best-practices/v/1.0.0
# standards/pci-dss/v/3.2.1
```

Reply:
This info is already in the variable description, so removing from here so we don't have duplicate places to maintain

9d564a68e95a907d1eec5bc4e2a0913516ec0784

Reply:
3d128cb09847d674b7b177a4fe158d87036330be